stdlib/stat.pyi

```diff
@@ -14,81 +15,84 @@ def S_ISPORT(mode: int) -> int: ...
+S_IFLNK: Literal[0o120000]
+S_IFREG: Literal[0o100000]
+S_IFSOCK: Literal[0o140000]
+S_IFBLK: Literal[0o060000]
+S_IFCHR: Literal[0o020000]
+S_IFDIR: Literal[0o040000]
 
-S_IRWXU: int
-S_IRUSR: int
-S_IWUSR: int
-S_IXUSR: int
+S_IFDOOR: Literal[0]
+S_IFPORT: Literal[0]
+S_IFWHT: Literal[0]
 
-S_IRWXG: int
-S_IRGRP: int
-S_IWGRP: int
-S_IXGRP: int
+S_ISUID: Literal[0o4000]
+S_ISGID: Literal[0o2000]
+S_ISVTX: Literal[0o1000]
 
-S_IRWXO: int
-S_IROTH: int
-S_IWOTH: int
-S_IXOTH: int
+S_IRWXU: Literal[0o0700]
+S_IRUSR: Literal[0o0400]
+S_IWUSR: Literal[0o0200]
+S_IXUSR: Literal[0o0100]
 
-S_ENFMT: int
-S_IREAD: int
-S_IWRITE: int
-S_IEXEC: int
+S_IRWXG: Literal[0o0070]
+S_IRGRP: Literal[0o0040]
+S_IWGRP: Literal[0o0020]
+S_IXGRP: Literal[0o0010]
 
-UF_NODUMP: int
-UF_IMMUTABLE: int
-UF_APPEND: int
-UF_OPAQUE: int
-UF_NOUNLINK: int
-UF_COMPRESSED: int  # OS X 10.6+ only
-UF_HIDDEN: int  # OX X 10.5+ only
-SF_ARCHIVED: int
-SF_IMMUTABLE: int
-SF_APPEND: int
-SF_NOUNLINK: int
-SF_SNAPSHOT: int
+S_IRWXO: Literal[0o0007]
+S_IROTH: Literal[0o0004]
+S_IWOTH: Literal[0o0002]
+S_IXOTH: Literal[0o0001]
 
-FILE_ATTRIBUTE_ARCHIVE: int
-FILE_ATTRIBUTE_COMPRESSED: int
-FILE_ATTRIBUTE_DEVICE: int
-FILE_ATTRIBUTE_DIRECTORY: int
-FILE_ATTRIBUTE_ENCRYPTED: int
-FILE_ATTRIBUTE_HIDDEN: int
-FILE_ATTRIBUTE_INTEGRITY_STREAM: int
-FILE_ATTRIBUTE_NORMAL: int
-FILE_ATTRIBUTE_NOT_CONTENT_INDEXED: int
-FILE_ATTRIBUTE_NO_SCRUB_DATA: int
-FILE_ATTRIBUTE_OFFLINE: int
-FILE_ATTRIBUTE_READONLY: int
-FILE_ATTRIBUTE_REPARSE_POINT: int
-FILE_ATTRIBUTE_SPARSE_FILE: int
-FILE_ATTRIBUTE_SYSTEM: int
-FILE_ATTRIBUTE_TEMPORARY: int
-FILE_ATTRIBUTE_VIRTUAL: int
+S_ENFMT: Literal[0o2000]
+S_IREAD: Literal[0o0400]
+S_IWRITE: Literal[0o0200]
+S_IEXEC: Literal[0o0100]
+
+UF_APPEND: Literal[0x00000004]
+UF_COMPRESSED: Literal[0x00000020]  # OS X 10.6+ only
+UF_HIDDEN: Literal[0x00008000]  # OX X 10.5+ only
+UF_IMMUTABLE: Literal[0x00000002]
+UF_NODUMP: Literal[0x00000001]
+UF_NOUNLINK: Literal[0x00000010]
+UF_OPAQUE: Literal[0x00000008]
+
+SF_APPEND: Literal[0x00040000]
+SF_ARCHIVED: Literal[0x00010000]
+SF_IMMUTABLE: Literal[0x00020000]
+SF_NOUNLINK: Literal[0x00100000]
+SF_SNAPSHOT: Literal[0x00200000]
+
+FILE_ATTRIBUTE_ARCHIVE: Literal[32]
+FILE_ATTRIBUTE_COMPRESSED: Literal[2048]
+FILE_ATTRIBUTE_DEVICE: Literal[64]
+FILE_ATTRIBUTE_DIRECTORY: Literal[16]
+FILE_ATTRIBUTE_ENCRYPTED: Literal[16384]
+FILE_ATTRIBUTE_HIDDEN: Literal[2]
+FILE_ATTRIBUTE_INTEGRITY_STREAM: Literal[32768]
+FILE_ATTRIBUTE_NORMAL: Literal[128]
+FILE_ATTRIBUTE_NOT_CONTENT_INDEXED: Literal[8192]
+FILE_ATTRIBUTE_NO_SCRUB_DATA: Literal[131072]
+FILE_ATTRIBUTE_OFFLINE: Literal[4096]
+FILE_ATTRIBUTE_READONLY: Literal[1]
+FILE_ATTRIBUTE_REPARSE_POINT: Literal[1024]
+FILE_ATTRIBUTE_SPARSE_FILE: Literal[512]
+FILE_ATTRIBUTE_SYSTEM: Literal[4]
+FILE_ATTRIBUTE_TEMPORARY: Literal[256]
+FILE_ATTRIBUTE_VIRTUAL: Literal[65536]
 
 if sys.platform == "win32" and sys.version_info >= (3, 8):
     IO_REPARSE_TAG_SYMLINK: int
```

Comment:
Can you change these to `Literal`s too? You can find the right values by setting them all to `Literal[123]` and looking at stubtest errors.

Reply:
Sorry, I don't have a windows machine 😞 
It is not clear what these are from their definition: https://github.com/python/cpython/blob/f4c03484da59049eb62a9bf7777b963e2267d187/Modules/_stat.c#L595-L606

Maybe in the next PR?